cmd/flags_assigners.go

```diff
@@ -379,6 +379,37 @@ func assignCreateGroupOptions(cmd *cobra.Command) (*gitlab.CreateGroupOptions, e
 	return &opts, nil
 }
 
+// assignAddProjectMemberOptions assigns the flags' values to gitlab.AddProjectMemberOptions fields.
+// If a flag's default value is not changed by the caller,
+// it's value will not be assigned to the associated gitlab.AddProjectMemberOptions field.
+func assignAddProjectMemberOptions(cmd *cobra.Command) (*gitlab.AddProjectMemberOptions, error) {
+	var opts gitlab.AddProjectMemberOptions
+
+	if f := cmd.Flag("access-level"); f != nil {
+		opts.AccessLevel = gitlab.AccessLevel(getFlagAccessLevel(cmd, "access-level"))
+	}
+
+	return &opts, nil
+}
+
+// assignAddGroupMemberOptions assigns the flags' values to gitlab.AddGroupMemberOptions fields.
+// If a flag's default value is not changed by the caller,
+// it's value will not be assigned to the associated gitlab.AddGroupMemberOptions field.
+func assignAddGroupMemberOptions(cmd *cobra.Command) (*gitlab.AddGroupMemberOptions, error) {
+	var opts gitlab.AddGroupMemberOptions
+
+	if f := cmd.Flag("access-level"); f != nil {
+		opts.AccessLevel = gitlab.AccessLevel(getFlagAccessLevel(cmd, "access-level"))
+	}
+	if f := cmd.Flag("expires-at"); f != nil {
```

Comment:
No need to check if this is nil since this is meant to be assigned on `new command` `init func`. Just check if it is changed then assign the value to the struct field.